Makefile

```
@@ -6,6 +6,12 @@ CONTROLLER_ENV_GID ?= $(shell id -g)
 MANAGER_ENV_UID ?= $(shell id -u)
 MANAGER_ENV_GID ?= $(shell id -g)
 
+ifeq ($(shell test -e .env.local && echo -n yes),yes)
+	COMPOSE_ENV_FILE ?= .env.local
+else 
+	COMPOSE_ENV_FILE ?= .env
+endif
+

```

Less verbose version:
```suggestion
COMPOSE_ENV_FILE = $(firstword $(wildcard .env.local .env))
```